tsconfig.json

```diff
@@ -1,14 +1,18 @@
 {
   "compilerOptions": {
     "jsx": "react",
-    "lib": ["es2015", "dom"],
+    "target": "ES2021", // Since lowest browser support is for Safari 14
+    "lib": ["DOM", "DOM.iterable"],
     "noImplicitAny": false, // TODO: Enable once all existing code is typed
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "strictNullChecks": true,
+    "strictPropertyInitialization": true, //This option requires `--strictNullChecks` be enabled in order to take effect
+    "forceConsistentCasingInFileNames": true,
     "esModuleInterop": true,
     "allowJs": true,
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "moduleResolution": "node"
```

Comment:
this is looking good - just a bit of a nitpicky one, it would be good to order these configs alphabetically (easier to find the values at a glance & also add consistent spacing to the comments.


Here is what my editor's formatter output.

```json
  "compilerOptions": {
    "allowJs": true,
    "downlevelIteration": true,
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "jsx": "react",
    "lib": ["DOM", "DOM.iterable"],
    "moduleResolution": "node",
    "noImplicitAny": false, // TODO: Enable once all existing code is typed
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "strictNullChecks": true,
    "strictPropertyInitialization": true, // Requires `--strictNullChecks` be enabled in order to take effect
    "target": "ES2021" // Since lowest browser support is for Safari 14
  },
```

I also simplified the comment on `strictPropertyInitialization`